internal/providers/terraform/aws/testdata/spot_instance_request_test/spot_instance_request_test.golden

```diff
@@ -0,0 +1,13 @@
+
+ Name                                             Monthly Qty  Unit   Monthly Cost 
+                                                                                   
+ aws_spot_instance_request.spot_instance_1                                         
+ └─ Instance usage (Linux/UNIX, spot, m3.medium)          730  hours         $4.89 
+                                                                                   
+ aws_spot_instance_request.spot_instance_2                                         
+ └─ Instance usage (Linux/UNIX, spot, m3.large)           730  hours        $22.85 
```

Comment:
Sadly this method won't work, as the spot prices change often and will cause the tests to fail. Instead you'll need to test against the price hash. See here: https://github.com/infracost/infracost/blob/59b89e4f850b9f9f1e7930cb2e191549555478c0/internal/providers/terraform/aws/autoscaling_group_test.go#L23 as an example of how to do this.

Reply:
Yeah, no problem. 
I was just waiting for the github pipeline to go back so I can continue, thank you for the help!

Reply:
done